File: Workshop/maven-java-playground/src/test/java/HelloWorldTest.java
import org.junit.jupiter.api.Test;

import static org.assertj.core.api.Assertions.assertThat;

public class HelloWorldTest {

    @Test
    public void testDoSomething() {
        final String amazingString = "MyAmazingString";

        final String result = HelloWorld.doSomething(amazingString);

        assertThat(result).isEqualTo("MYAMAZINGSTRING");
    }

}

File: kubernetes/k8s_001/oxygen/src/test/kotlin/de/roskenet/oxygen/HelloWorldTest.kt
package de.roskenet.oxygen

import de.roskenet.oxygen.helloworld.HelloWorld
import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.Test

class HelloWorldTest {

    val helloWorld = HelloWorld("Servus, %s!")

    @Test
    fun `HelloWorld with a name`() {
        val (greeting, isHappy) = helloWorld.returnSomething("Elvis")
        assertThat(greeting).isEqualTo("Servus, Elvis!")
        assertThat(isHappy).isTrue()
    }
}